addon/components/o-s-s/currency-input.ts

```diff
@@ -31,6 +32,14 @@ export default class OSSCurrencyInput extends Component<OSSCurrencyInputArgs> {
     return this.selectedCurrency.symbol || '—';
   }
 
+  get selectedCurrencyCode(): string {
+    return this.selectedCurrency.code || '...';
```

Comment:
I doubt that we put the '...' when no value is selected ; please check with Victor.